go-controller/pkg/ovn/secondary_layer2_network_controller.go

```diff
@@ -850,5 +850,20 @@ func (oc *SecondaryLayer2NetworkController) updateLocalPodEvent(pod *corev1.Pod)
 }
 
 func (oc *SecondaryLayer2NetworkController) reconcileLiveMigrationTargetZone(kubevirtLiveMigrationStatus *kubevirt.LiveMigrationStatus) error {
+	// Only primary networks has a gateway to reconcile
+	if !oc.IsPrimaryNetwork() {
+		return nil
+	}
+	networkID, err := oc.getNetworkID()
```

Comment:
you can use `GetNetworkID` which won't do parsing and will just get the cached value on NetInfo itself